pdc/apps/componentbranch/filters.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright (c) 2017 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+import django_filters
+from django.db.models import Q
+from datetime import datetime
+
+from pdc.apps.common.filters import CaseInsensitiveBooleanFilter
+from pdc.apps.componentbranch.models import (
+    ComponentBranch, SLA, SLAToComponentBranch)
+
+
+def string_to_bool(item):
```

Comment:
We have a similar method convert_str_to_bool in file apps/common/hacks.py
Maybe we can think about to merge them?